.eslintrc.js

```diff
@@ -148,6 +148,15 @@ module.exports = {
       'no-prototype-builtins': 'off',
       'no-multi-spaces': 'off',
     },
+  }, {
```

Comment:
Can you un-ignore the dev folder here? I think it shouldn't add too much noise to this PR. Otherwise make a separate PR  for that

Reply:
`update-authors.js` is already un-ignored, `sscce-helpers.js` already follows the code style properly, and this PR modifies every other js/ts file present in dev. I agree it's time to un-ignore the dev folder 🥳